### ISSUE TITLE
consentform.html localization

### ISSUE THREAD

**Issue by rafaellop** [tobyherrmann/cordova-plugin-google-consent#9]
When working on the consent implementation I noticed there's no way to translate the `constentform.html` which is generated during the build. My final conclusion regarding the GDPR regulations is that there's no real sense in putting the consent form as it is now and using the plugin while it doesn't allow to show the consent information in simple and *understandable* form to app users from the EEU. There are many countries in EU that are not English-speaking. So presenting English information for non-English people doesn't comply with the law and it is indeed useless for developers who want to be in accordance with the regulations. Or maybe there's a way to localize the consent form provided by google?

**Comment by 7h3m4dS0u1** [comment]
Hi @rafaellop , I have the same problem. Do you find any solution?
Thanks

**Comment by rafaellop** [comment]
Yes, I removed the plugin and create my own implementation which doesn't use the Google functions.

**Comment by 7h3m4dS0u1** [comment]
Ok. Thanks.

**Comment by cmartinezv** [comment]
+1
